<launch>
    <!-- Set 3d camera maximal relevant distance in meters -->
    <param name="but_server_node/max_range" type="double" value="4.5"/>

    <!-- Other parameters -->
    <param name="but_server_node/pointcloud_min_z" value="0.0" type="double"/>
    <!--param name="but_server_node/use_old_im" value="true" type="bool"/-->
    <!--param name="but_server_node/resolution" value="0.025" type="double"/-->
    <param name="but_server_node/resolution" value="0.05" type="double"/>
    <param name="but_server_node/frame_skip" value="5" type="int"/>

    <!-- Run our dynamic scene model server node -->
    <node name="but_server_node" pkg="srs_env_model" type="but_server_node">
	    <param name="diferential_frames_count" value="1" type="int"/> 
		<remap from="cpc_camera_info" to="/cam3d/rgb/camera_info" />
		<remap from="ocp_camera_info" to="/cam3d/rgb/camera_info" /> 
		<remap from="pcp_points" to="/cam3d/rgb/points" />
    </node>
</launch>
